build/ci/ansible/roles/kubefate/files/clean.sh

```diff
@@ -1,18 +1,35 @@
 #/bin/bash
 
-clean_when_timeout()
+clean()
 {
   rm -rf ${BASE_DIR}/*
 
-  # Install Kind
+  # Delete kind
```

Comment:
This is not kind deletion, should replace with remove k8s cluster

Reply:
done